lib/config/validation.spec.ts

```diff
@@ -163,10 +163,15 @@ describe('config/validation', () => {
             matchCurrentValue: '/^2/i',
             enabled: true,
           },
+          {
+            matchPackageNames: ['bad'],
+            matchNewValue: '/^2(/',
+            enabled: true,
+          },
         ],
       };
       const { errors } = await configValidation.validateConfig('repo', config);
-      expect(errors).toHaveLength(2);
+      expect(errors).toHaveLength(1);
```

Comment:
Why did it go down?

Reply:
All previous "invalid" - non-regex - values became valid